plugin/pkg/scheduler/scheduler.go

```diff
@@ -124,7 +124,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {		
```

Comment:
The "E2eSchedulingLatency" should be calculated after the "BindingLatency" operation, by the "**defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))**". While in fact, the "E2eSchedulingLatency" is calculated before the "BindingLatency".  There may be some optimization for code compile